src/otx/algo/detection/heads/custom_anchor_generator.py

```diff
@@ -47,7 +47,7 @@ def gen_base_anchors(self) -> list[torch.Tensor]:
                 center=torch.Tensor(centers),
             )
             multi_level_base_anchors.append(base_anchors)
-        return multi_level_base_anchors
+        self.base_anchors = multi_level_base_anchors
```

Comment:
Why we need this change?

Reply:
To change head's anchor
Previous code:
`head.base_anchors = head.gen_base_anchors()`
Now
`head.gen_base_anchors()`